src/megaapi_impl.cpp

```diff
@@ -2609,11 +2609,6 @@ void MegaApiImpl::addEntropy(char *data, unsigned int size)
     if(PrnGen::rng.CanIncorporateEntropy())
```

Comment:
It's a good idea to *always* put curly brackets around all conditional blocks, even if they're just a single line.